modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/exportimport/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -184,7 +187,22 @@ null, _className, _getBytes(_fileName, inputStream), "JSON",
 				).build(),
 				_taskItemDelegateName);
 
-		_batchEngineImportTaskExecutor.execute(batchEngineImportTask);
+		try {
+			BatchEngineImportTask finalBatchEngineImportTask =
+				batchEngineImportTask;
+
+			TransactionInvokerUtil.invoke(
+				transactionConfig,
+				() -> {
+					_batchEngineImportTaskExecutor.execute(
```

Comment:
Whould we create this transaction inside the Batch Engine method? I am wondering if this issue is also reproducible if we creates the Batch Import task from a Bundle.

Could you verify @DaniSzimko please?